bucket/gog-galaxy.json

```diff
@@ -0,0 +1,43 @@
+{
+  "version": "2.0.25.2",
+  "description": "Connect GOG GALAXY 2.0 with multiple platforms and unite all your games and friends scattered across them in one powerful app (A game platform designed to connect all game platforms).",
+  "homepage": "https://www.gog.com/galaxy",
+  "license": {
+    "identifier": "Proprietary",
+    "url": "https://support.gog.com/hc/en-us/articles/212632089-User-Agreement"
+  },
+  "depends": "innounp",
+  "url": "https://content-system.gog.com/open_link/download?path=/open/galaxy/client/setup_galaxy_2.0.25.2.exe",
+  "hash": "md5:d09244d6ccb563da59ffd83767b50b2b",
+  "installer": {
+    "script": [
+      "$file = \"$dir\\$fname\"",
+      "Expand-InnoArchive -Path $file -ExtractDir '{%ALLUSERSPROFILE}' -DestinationPath $env:ALLUSERSPROFILE",
```

Comment:
Can you remove this? Since it does not satisfy the concept of the bucket `All apps in this repository should NOT require elevated privileges while installation.`.

Reply:
This instruction did not cause elevated privileges, actually, at least there's no pop-up window for elevated privileges.
There are four directories after unpacking with innounp: `{%ALLUSERSPROFILE}`, `{app}`, `{fonts}`, `{tmp}`.
Where `{%ALLUSERSPROFILE}` is designed to be under the `$env:ALLUSERSPROFILE` path (`C:\ProgramData`)
If the path is not organized in this way, the program will report an error:
`Essential components needed to start GOG Galaxy are missing or incorrectly configured. if this continues, please reinstall the application.`
I didn't find a solution, I think that's how they write the program.

Reply:
> This instruction did not cause elevated privileges, actually, at least there's no pop-up window for elevated privileges.

`C:\ProgramData` requires the Administrators role to write, you'll not see UAC if you're the Administrator user.

Reply:
To check current user whether is Administrator or not:
```powershell
[System.Security.Principal.WindowsIdentity]::GetCurrent().Name -in (Get-LocalGroupMember -Group "Administrators").Name
```

Reply:
You are right, I'm the administrator.
So what should we do? GOG will read components in `$env:ALLUSERSPROFILE`, the program cannot accept a parameter to change this behavior.